shopping-cart/app/shoppingcart/common/ExtendedLineItemBean.java

```diff
@@ -0,0 +1,36 @@
+package shoppingcart.common;
+
+import common.contexts.UserContext;
+import common.controllers.ReverseRouter;
+import common.models.AttributeBean;
+import common.models.LineItemBean;
+import common.models.ProductDataConfig;
+import io.sphere.sdk.carts.LineItem;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+public class ExtendedLineItemBean extends LineItemBean {
```

Comment:
Okay so I renamed the simplified version as "MiniCartLineItem" and the extended version as "CartLineItem". I think it makes sense that a cart object extends a mini cart object, since they are basically the same but with more/less info.